multisig/simulations/vip-004/vip-004-opbnbtestnet/simulations.ts

```diff
@@ -58,7 +57,7 @@ forking(16716512, () => {
     });
 
     it("Should set correct reward token address", async () => {
-      const isActive = await xvsStore.rewardTokens(XVS);
+      const isActive = await xvsStore.rewardTokens(opbnbtestnet.XVS);
```

Comment:
There are more uses of `XVS` in this file, to be replaced with `opbnbtestnet.XVS`. After doing that, we could remove the constant `XVS`